src/main/scala/com/databricks/spark/sql/perf/mllib/fpm/ItemSetGenerator.scala

```diff
@@ -0,0 +1,60 @@
+package com.databricks.spark.sql.perf.mllib.fpm
```

Comment:
Can you move this file to perf.mllib.data with the other data generation code?  (But I like that you have it as a separate file.)